### PR TITLE
Update Icons Management

### PR DESCRIPTION
**Pull request by talarian1**

### CODE REVIEW

File: iconsmanager.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+	case "critical":
+		return criticalSeveritySource
+
+	case "high":
+		return highSeveritySource
+
+	case "medium":
+		return mediumSeveritySource
+
```
Comment (yahavi) [inline iconsmanager.go:30]: Redundant spaces
```suggestion
	case "critical":
		return criticalSeveritySource
	case "high":
		return highSeveritySource
	case "medium":
		return mediumSeveritySource
```

File: iconsmanager.go [hunk 2]
```diff
@@ -0,0 +1,35 @@
+package main
+
+import "fmt"
+
+const (
+	baseSourceUrl               = "https://raw.githubusercontent.com/jfrog/frobot/master/resources/"
```
Comment (yahavi) [inline iconsmanager.go:6]:
```suggestion
	baseResourceUrl               = "https://raw.githubusercontent.com/jfrog/frobot/master/resources/"
```

File: iconsmanager.go [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+	noVulnerabilityBannerSource = "noVulnerabilityBanner.png"
+	vulnerabilitiesBannerSource = "vulnerabilitiesBanner.png"
+	criticalSeveritySource      = "criticalSeverity.png"
+	highSeveritySource          = "highSeverity.png"
+	mediumSeveritySource        = "mediumSeverity.png"
+	lowSeveritySource           = "lowSeverity.png"
```
Comment (yahavi) [inline iconsmanager.go:12]: I think these consts should be grouped together in a new type. For example:
```go
type imageSource string

const (
	baseSourceUrl                           = "https://raw.githubusercontent.com/jfrog/frobot/master/resources/"
	noVulnerabilityBannerSource imageSource = "noVulnerabilityBanner.png"
	vulnerabilitiesBannerSource imageSource = "vulnerabilitiesBanner.png"
	criticalSeveritySource      imageSource = "criticalSeverity.png"
	highSeveritySource          imageSource = "highSeverity.png"
	mediumSeveritySource        imageSource = "mediumSeverity.png"
	lowSeveritySource           imageSource = "lowSeverity.png"
)

 func GetIconTag(imageSource imageSource) string {

```

File: main.go [hunk 4]
```diff
@@ -22,7 +22,8 @@ import (
 	"github.com/urfave/cli"
 )
 
-const commandHelpTemplate string = `{{.HelpName}}{{if .UsageText}}
+const (
+	commandHelpTemplate = `{{.HelpName}}{{if .UsageText}}
```
Comment (yahavi) [inline main.go:26]: We don't really need those templates. Looks how elegant is build-info-go with urfave v2: https://github.com/jfrog/build-info-go/blob/main/main.go

**Review by yahavi — APPROVED** [review]

File: main.go (hunk 4 above)
Comment (talarian1) [reply main.go]: Looks great! :)

File: iconsmanager.go [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+import (
+	"fmt"
+	"strings"
+)
+
+type imageSource string
+
+const (
+	baseResourceUrl                         = "https://raw.githubusercontent.com/jfrog/frobot/master/resources/"
```
Comment (yahavi) [inline iconsmanager.go:11]:
```suggestion
	baseResourceUrl                         = "https://raw.githubusercontent.com/jfrog/frogbot/master/resources/"
```

**Review by yahavi — CHANGES_REQUESTED** [review]

File: .github/workflows/frogbot.yml [hunk 6]
```diff
@@ -0,0 +1,34 @@
+
```
Comment (yahavi) [inline .github/workflows/frogbot.yml:1]:
```suggestion
```

**Pull request merged**